docs/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Ignore the generated api docs and auto-build docs
```

Comment:
Woo, another blunder on my part. Forgot to include the doc changes for GitIgnore. Good catch.